### PR TITLE
3646 html css to image

### PR DESCRIPTION
**Pull request by alysonturing**
New actions
- Create Image From HTML
- Create Image From URL

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/3973"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/html_css_to_image/actions/create-image-from-url/create-image-from-url.mjs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import htmlCssToImageApp from "../../html_css_to_image.app.mjs";
+
+export default {
+  key: "html_css_to_image-create-image-from-url",
+  name: "Create Image From URL",
+  description: "Create Image From URL. [See the docs](https://docs.htmlcsstoimage.com/getting-started/using-the-api/#creating-an-image).",
+  version: "0.0.6",
```
Comment (andrewjschuang) [inline components/html_css_to_image/actions/create-image-from-url/create-image-from-url.mjs:7]:
```suggestion
  version: "0.0.1",
```

**Review by andrewjschuang — COMMENTED** [review]
@alysonturing thanks for the contribution!

I've left a comment below, but otherwise LGTM! Moving to QA

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[HTML_URL_to_image.pdf](https://github.com/PipedreamHQ/pipedream/files/9307897/HTML_URL_to_image.pdf)

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**